pkg/tsdb/cloudwatch/cloudwatch_integration_test.go

```diff
@@ -47,7 +54,11 @@ func Test_CloudWatch_CallResource_Integration_Test(t *testing.T) {
 	}
 
 	im := datasource.NewInstanceManager(func(s backend.DataSourceInstanceSettings) (instancemgmt.Instance, error) {
-		return DataSource{Settings: models.CloudWatchSettings{}}, nil
+		return DataSource{Settings: models.CloudWatchSettings{
+			AWSDatasourceSettings: awsds.AWSDatasourceSettings{
+				Region: "us-east-1",
```

Comment:
There's something about this "us-east-1" here and the assertions below that may be misleading.
For example you can put any string like "some default region" here and the tests will still pass because the regions are coming from a constant list.
What could be interesting would be to show that when Region is omitted here (len == 0 or Region == ""), then you assert a 400 status, bad request. May be able to assert the error message as well?
I believe this is what your PR adds.

Reply:
We have that captured in Regions_test but I can put that here too. Wasn't sure if we only test happy path use cases for our integration tests...but that seems silly, we should test for known errors!